samples/tmart/tmart-portal/main.bal

```diff
@@ -0,0 +1,52 @@
+import ballerina/log;
+import ballerina/http;
+import ballerinax/jaeger as _;
+
+type POrder record {
+    string orderId;
+    string customerId;
+    string productId;
+    int quantity;
+};
+
+configurable string shipmentHost = "localhost";
+configurable string inventoryHost = "localhost";
+configurable string crmHost = "localhost";
+configurable string bookParkHost = "localhost";
+
+public function main() {
+    log:printInfo("Starting the TMART portal...");
+}
+
+service /tmart on new http:Listener(9100){
+    resource function post orders(POrder porder) returns json|error {
+        log:printInfo("Order received: " + porder.orderId);
+        do {
+            http:Client customers = check new("http://" + crmHost + ":9102/customers");
+            string address = check customers->/customers/[porder.customerId]/address;
+
+            http:Client inventory = check new("http://" + inventoryHost + ":9103/inventory");
```

Comment:
shouldn't we use below way as usually host+port is the one we need to configure together?
```configurable string inventoryHost = "http://localhost:9103/inventory"
http:Client inventory = check new(inventoryHost);```